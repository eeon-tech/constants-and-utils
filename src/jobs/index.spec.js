const jobs = require('./index')

describe('jobs', () => {
  describe('cleanPeriods', () => {
    it('should remain consistent', () => {
      expect(jobs.cleanPeriods).toMatchSnapshot()
    })
  })

  describe('timezone', () => {
    it('should remain consistent', () => {
      expect(jobs.timezone).toMatchSnapshot()
    })
  })

  describe('queueNames', () => {
    it('should remain consistent', () => {
      expect(jobs.queueNames).toMatchSnapshot()
    })
  })
})
