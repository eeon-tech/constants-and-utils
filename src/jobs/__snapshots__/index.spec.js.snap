// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jobs cleanPeriods should remain consistent 1`] = `
Object {
  "1D": 86400,
}
`;

exports[`jobs queueNames should remain consistent 1`] = `
Object {
  "calculateDerivedSecurityMetrics": "calculateDerivedSecurityMetrics",
  "calculateEEONPerformanceScores": "calculateEEONPerformanceScores",
  "calculateMacroIndicators": "calculateMacroIndicators",
  "calculateMarketInsights": "calculateMarketInsights",
  "calculateSecurityInsights": "calculateSecurityInsights",
  "collectFinancialStatementsUponEarnings": "collectFinancialStatementsUponEarnings",
  "primeCache": "primeCache",
  "runPreMarketJobs": "runPreMarketJobs",
  "sendRealtimeSecurities": "sendRealtimeSecurities",
  "sendUserSecurityAlertMessages": "sendUserSecurityAlertMessages",
  "triggerUserSecurityAlerts": "triggerUserSecurityAlerts",
}
`;

exports[`jobs timezone should remain consistent 1`] = `"America/New_York"`;
